test/irb/test_helper.rb

```diff
@@ -0,0 +1,14 @@
+module IRBTestHelper
```

Comment:
I name it like this because:

- `TestHelper` is too general and could be used in other lib's tests.
- `IRB::TestHelper` adds a new constant under the `IRB` constant, which ideally shouldn't affect anything, but I want to avoid unless necessary.